R/utils-validation_classes.R

```diff
@@ -0,0 +1,24 @@
+validate_internal_class <- function(
```

Comment:
This was just moved to it's own file from a different file so no need for review. The only thing added was the `"target_validations"` as an option of class to validate elements of an object as